libbeat/docs/shared-autodiscover.asciidoc

```diff
@@ -244,7 +244,8 @@ endif::[]
  `add_resource_metadata` can be done for `node` or `namespace`. By default all labels will be included
   while annotations are not added by default. This settings are useful when labels' and annotations'
   storing requires special handling to avoid overloading the storage output. The enrichment of `node` or `namespace` metadata
-  can be individually disabled by setting `enabled: false`.
+  can be individually disabled by setting `enabled: false`. If resource is `pod` and it is created from a `deployment`, by default
+  the deployment name is added, this can be disabled by set to `false`.
```

Comment:
```suggestion
  the deployment name is added, this can be disabled by setting `deployment: false`.
```
similar to working in https://github.com/elastic/beats/blob/main/libbeat/docs/shared-autodiscover.asciidoc